lnwallet/rpcwallet/sign_coordinator.go

```diff
@@ -0,0 +1,1039 @@
+package rpcwallet
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnrpc/signrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/walletrpc"
+	"google.golang.org/grpc"
+)
+
+var (
+	// ErrRequestTimeout is the error that's returned if we time out while
+	// waiting for a response from the remote signer.
+	ErrRequestTimeout = errors.New(
+		"remote signer response timeout reached")
```

Comment:
nit: closing parenthesis on new line, since this isn't using a formatting directive (e.g. `Errorf()`)?